Streetcode/Streetcode.WebApi/Configuration/RoleAndUserConfiguration.cs

```diff
@@ -57,8 +64,11 @@ private static async Task AddRolesAsync(IServiceProvider services)
         private static async Task AssignRole(IServiceProvider services, string email, string role)
         {
             UserManager<User> userManager = services.GetService<UserManager<User>>() !;
-            User user = await userManager!.FindByEmailAsync(email);
-            await userManager.AddToRoleAsync(user, role);
+            User? user = await userManager!.FindByEmailAsync(email);

```

Comment:
`UserManager<User> userManager` is already non nullable so I think the null forgiving operator in this case `await userManager!.FindByEmailAsync(email)` is redundant

Reply:
Yes. you are right. Thanks for proposed adjustments.